docker/elasticsearch/7/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM blacktop/elasticsearch:7.10.1
+FROM docker.elastic.co/elasticsearch/elasticsearch:7.12.1
```

Comment:
We were using `blacktop/elasticsearch` for a reason. Until we dig up what that was, I'm going to hold off on approving this PR.

Reply:
Looks like because it was Alpine-based: https://github.com/cfpb/consumerfinance.gov/pull/6052

Is `docker.elastic.co/elasticsearch/elasticsearch:7.12.1` Alpine-based?

Reply:
Looks like they're CentOS 8-based: https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html

Reply:
No, the elasticsearch image is not Alpine-based, it uses centos as the base image:
https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html

Reply:
So what do we want to do about this? Would blacktop's 7.10.2 be enough to satisfy Twistlock? Or should we build our own?

Reply:
I do not believe that upgrading blacktop from 7.10.1 to 7.10.2 would be enough to satisfy Twistlock.
Why can we not use CentOS 8-based docker images in a development environment?
In the production environment the AWS Elasticsearch service is being used.
We could build our own Alpine-based elasticsearch 7.12 docker image or create a PR to update blacktop to 7.12.